ballerina/Ballerina.toml

```
@@ -5,12 +5,12 @@ name = "hubspot.marketing.subscriptions"
 version = "1.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = ["hubspot","marketing","subscriptions"] 
+icon = "../docs/setup/resources/icon.png" 
```

Let move the icon out of docs directory and add it under ballerina directory
```suggestion
icon = "icon.png" 
```